sparkle.js

```diff
@@ -22,15 +23,23 @@ window.onload = function() {
   resetState();
 
   function displayPixelatedImage() {
+    if(!canvas.width || !canvas.height) {
+      return;
+    }
```

Comment:
Before seeing this PR, i removed the initial call to `resetState` so I don't think this is needed anymore?